build.cake

```diff
@@ -50,11 +50,7 @@ Setup(() =>
 //////////////////////////////////////////////////////////////////////
 
 Task("Clean")
-    .Does(() =>
-{
-    CleanDirectories(new DirectoryPath[] {
-        buildResultDir, binDir, testResultsDir, nugetRoot});
-});
+    .Does(() => CleanDirectories(parameters.Paths.Directories.ToClean));
```

Comment:
@devlead hmm, this looks _wrong_ to me.  I would prefer to still include the {}'s, but that might just be a personal preference.